megalinter/descriptors/bash.megalinter-descriptor.yml

```diff
@@ -7,10 +7,17 @@ descriptor_flavors:
   - c_cpp
   - formatters
 file_extensions:
+  - ""
```

Comment:
What is the impact of this in projects?

Reply:
If the file without an extension does not include a matching shebang, my understanding is that there will be [no impact](https://github.com/oxsecurity/megalinter/blob/d5670fa4d09893729c81131fb6e71f15790e3770/megalinter/utils.py#L206-L208)

Reply:
However, this is needed in order for the conditional logic to even reach the point where it checks for matches in `file_contains_regex`